src/icalendar/tests/test_unit_prop.py

```diff
@@ -518,6 +500,38 @@ def test_prop_TypesFactory(self):
         )
 
 
+
+vDDDTypes_list = [
+    vDDDTypes(pytz.timezone('US/Eastern').localize(datetime(year=2022, month=7, day=22, hour=12, minute=7))),
```

Comment:
I tried to add not too much. Hm. Since many of the arguments do not use a tz, that would create redundant tests.

Reply:
Could you explain how would they be redundant? I don't quite get that.

Reply:
A second or third case with timezone should do the job too. One with same timezone and one with different time zone but same instant in time.

Reply:
In ae9ac71 I added another time zone. I think, this covers the case:
- time zone differs
- time zone different from no time zone

Since no time zone is also a time zone in my view, all the cases would be covered. What do you think?